.changeset/dry-phones-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/typography': major
+---
+
+Updates `Body`, `Disclaimer`, `H1`, `H2`, `H3`, `InlineCode`, `InlineKeyCode`, `Link`, `Overline`, and `Subtitle` for dark mode refresh. Also add `Chart` component.
```

Comment:
I feel like calling the component `Chart` is a bit confusing. Would `ChartLabel` or something be more accurate?